modules/client-configuration/pages/clients-debian.adoc

```diff
@@ -26,15 +26,15 @@ Bootstrapping can be used with {debian} clients for performing initial state run
 
 Some preparation is required before you can register {debian} clients to the {productname} Server:
 
-* If you are using Debian{nbsp}9, install the ``apt-transport-https`` package on the client before you attempt to register.
+* If you are using Debian{nbsp}9, install the required packages on the client before you attempt to register.
   On the client, at the command prompt, as root, run:
 +
 ----
-apt install apt-transport-https
+apt install apt-transport-https python-apt python3-apt
```

Comment:
... and three here.

I'd keep "install the required packages" at line 29, and just the three packages at line 33, instead of duplicating the list.

Reply:
I ... did?

Reply:
Yes, you did. I think I am becoming daltonic or something. Either that or I need new glasses.

PR is fine, sorry about the noise (for some reason I thought you were adding `apt-transport-https` to line 29, when in fact you we are removing it.

Reply:
OK, I thought I was going mad for a bit there! Maybe go see an optometrist? :wink: